src/analytics/dto/filter-analytics-request-data.ts

```diff
@@ -0,0 +1,60 @@
+import { IsNotEmpty, IsOptional } from 'class-validator';
+import { Transform } from 'class-transformer';
+
+export class FilterAnalyticsRequestData {
+  @IsOptional()
+  browser?: string;
```

Comment:
```suggestion
  browser?: string | null;
```

do this for other as well

Reply:
updated